### DISCUSSION TITLE
NO FILE TO EXECUTE ON LATEST ASSET?

### DISCUSSION THREAD

**Discussion by adriamrtinez** [chris1111/Wireless-USB-Big-Sur-Adapter#271] [Q&A]
I'm running macOS 10.15.7 and I want to upgrade to Big Sur. However, I downloaded the latest asset for OpenCore but I don't know what should I do next. I have seen the video attached, but folder structure is completely different. Does anyone know what should I do to safely upgrade and get the wireless USB adapter working? 
Thanks!

**Comment by chris1111** [comment]
Sorry but Upgrade OpenCore is not discuss here

You have a lots of infos about to how to upgrade OpenCore in Google
